lib/fog/ovirt/models/compute/template.rb

```diff
@@ -42,11 +42,15 @@ def ready?
 
         def destroy(options = {})
           service.client.destroy_template(id)
+        rescue ::OVIRT::OvirtException => e
+          raise ::Fog::Ovirt::OvertError, e.message
         end
 
         def save
-          raise Fog::Errors::Error.new('Resaving an existing object may create a duplicate') if persisted?
+          raise ::Fog::Ovirt::Errors::OvirtError, 'Providing an existing object may create a duplicate object' if persisted?
           service.client.create_template(attributes)
+        rescue ::OVIRT::OvirtException => e
+          raise ::Fog::Ovirt::OvertError, e.message
```

Comment:
Typo, `ovErtError` instead of `Ovirt`

Reply:
fixed

Reply:
Could you add some tests to this? It looks like nobody covered that code path.